release notes/v0.54.0.md

```diff
@@ -0,0 +1,304 @@
+k6 `v0.54.0` is here 🎉! This release includes:
+
+- A new experimental CSV module
+- new ECMAScript features
+- Updated logo and branding
+
+## Breaking changes
+
+- [#3913](https://github.com/grafana/k6/pull/3913) changes mapping of the Golang's `math/big.Int` type to `bigint` type in k6.
+- [#3922](https://github.com/grafana/k6/pull/3922) removes `lib.Min` and `lib.Max` from k6's Go API, which could affect custom extensions that rely on these functions.
+- [#3838](https://github.com/grafana/k6/pull/3838) remove `k6/experimental/timers` - they are now available globally and no import is needed.
+- [#3944](https://github.com/grafana/k6/pull/3844) updates to `k6/experimental/websockets` made the default to binaryType to be `"blob"` as per the specification. Previously it was not set by default and required to be set and before that it was `"arraybuffer"`. With this change `k6/experimental/websockets` is now compliant with the specification.
```

Comment:
If I'm not mistaken previously it was always arraybuffer, I mean we had no case where default hasn't been set. But probably @mstoykov is the best person to chime in

Reply:
> , I mean we had no case where default hasn't been set.

In v0.53.0 it had no value and always required you to set it or receiving a binary message was an exception when it happened.

Reply:
```suggestion
- [#3944](https://github.com/grafana/k6/pull/3844) updates to `k6/experimental/websockets` made the binaryType default value equal to "blob". With this change, `k6/experimental/websockets` is now compliant with the specification.
```

Reply:
I think it might be better to simplify it more if we are not going to specify the exact way it changed over time. And it likely doesn't matter to most people.

Reply:
That sounds good to me. :shipit: